File: L2/PY/casa/5.py
i=0
while True:
    print("Insira os dados do {} funcionário: ".format(i),end=' ')
    x=list(map(float,input().split()))
    if x[0]==0:break
    x[0]-=(x[0]*0.13) if x[0]>=800 and x[0]<=1600 else 0
    x[0]-=(x[0]*0.22) if x[0]>1600 else 0
    x[0]+=(x[1]-160)*(0.5*(x[0]/160)) if x[1]>160 else 0
    print("O seu salário será {:.2f}R$".format(x[0]))
    i+=1
print('Até!')

File: L2/C/casa/5.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>
//
int main(){
    char*inp,*split;
    float v[2],novo;
    for(int f=0;;++f){
        printf("Insira os dados do %i funcionario: ",f);
        char*inp=(char*)malloc(1000*sizeof(char));
        fgets(inp,1000,stdin);
        if(inp[0]=='0'){
            break;
        }
        for(int i=0;split=strtok_r(inp," ",&inp);++i){
            v[i]=atof(split);
        }
        novo=v[0];
        1600>=novo&&novo>=800?novo-=(novo*0.13):novo>1600?novo-=(novo*0.22):novo;
        v[1]>160?novo+=(v[1]-160.0)*(0.5*(novo/160.0)):novo;
        printf("O seu salario sera: %.2fR$\n",novo);
    }
    printf("Verificacao check!\n");
}